metrics/internal/collectors/rbd-mirror.go

```diff
@@ -177,25 +145,6 @@ func (c *RBDMirrorCollector) Collect(ch chan<- prometheus.Metric) {
 						klog.Errorf("Failed to unmarshal description of image %q from site %q: %v", image.Name, site.SiteName, err)
 						continue
 					}
-					// LocalSnapshotTimestamp could be unavailable for a while during image resync.
```

Comment:
We can remove the entire if block and delete `internalcache.RBDMirrorPeerSiteDescription{}` structure in cache since we don't use that data anymore. Will save us some space.

https://github.com/red-hat-storage/ocs-operator/blob/68d03b12968375eb97651bc4b74cdf9b102c74ef/metrics/internal/collectors/rbd-mirror.go#L134-L149

Reply:
done